### DISCUSSION TITLE
install/commercial/deploy-requirements

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2326] [Announcements]
# install/commercial/deploy-requirements

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/deploy-requirements.html

**Comment by liyuerich** [comment]
网络拓扑 图片中使用了 Gloal Cluster， 是不是写错了？

应该是Global Cluster？

**Comment by liyuerich** [comment]
如果负载均衡模式是 metallb，则需要指定一个 VIP，供给 GLobal 集群的 insight 数据收集入口使用，子集群的 insight-agent 可上报数据到这个 VIP

GLobal 应该为 Global？

**Comment by windsonsea** [comment]
Fixed in PR #5286

**Reply by panpan0000** [reply]
from  @芳斌， 200个节点的长稳测试， Global的每台内存需要到32G都不太够。 所以硬件配置要跟着子集群的规模而相应增加。

**Reply by panpan0000** [reply]
突然发现我们硬件要求 没有磁盘空间和分区的要求
